src/PureScript/Backend/Chez/Convert.purs

```diff
@@ -63,9 +63,12 @@ codegenModule { name, bindings, exports, imports, foreign: foreign_ } =
         [ ImportSet $ ImportPrefix
             (ImportLibrary { identifiers: NEA.singleton "chezscheme", version: Nothing })
             "scm:"
+        , ImportSet $ ImportPrefix
+            (ImportLibrary { identifiers: NEA.cons' "_Chez_Runtime" [ "lib" ], version: Nothing })
```

Comment:
Warning: Bikeshedding ahead!

Since the library name has to match the file to which the identifier corresponds, is `_Chez_Runtime` what we want to call this? Or should it be named something more specific to PureScript? For example, 
`identifiers: NEA.cons' "purs" [ "backend", "chez-scheme", "lib" ]`

Reply:
That would then require the file to be placed under:
```
output/purs/backend/chez-scheme/lib.ss
```
Right?

Reply:
I believe so.

Reply:
Per Chez Scheme User Guide 9 (Section 2.4):
> For example, for base `/usr/lib/scheme`, library name `(app lib1)`, and extension `.sls`, the full path is `/usr/lib/scheme/app/lib1.sls`. So, if `(library-directories)` contains the pathnames `"/usr/lib/scheme/libraries"` and `"."`, and `(library-extensions)` contains the extensions `.ss` and `.sls`, the path of the `(tools sorting)` library must be one of the following.
> ```
> /usr/lib/scheme/libraries/tools/sorting.ss
> /usr/lib/scheme/libraries/tools/sorting.sls
> ./tools/sorting.ss
> ./tools/sorting.sls 
> ```